src/utilities.py

```diff
@@ -383,6 +385,26 @@ def get_victim(cli, nick, victim, in_chan, self_in_list=False, bot_in_list=False
         return
     return pl[pll.index(tempvictims.pop())] #convert back to normal casing
 
+#completes a partial role
+def complete_role(cli, nick, chan, role):
```

Comment:
utilities.py is deprecated; new things should go in functions.py and be user-aware. A nick completion function is already in functions.py as get_target() that you can base this function on.